backend/danswer/danswerbot/slack/listener.py

```diff
@@ -93,15 +82,11 @@ def prefilter_requests(req: SocketModeRequest, client: SocketModeClient) -> bool
             return False
 
         if (
-            msg in _SLACK_GREETINGS_TO_IGNORE
-            or remove_danswer_bot_tag(msg, client=client.web_client)
-            in _SLACK_GREETINGS_TO_IGNORE
+            req.payload.setdefault("event", {}).get("user", "")
```

Comment:
isn't this a user object? I dont think this gives back a string representing the ID